pkg/testrunner/testrunner.go

```diff
@@ -68,18 +69,17 @@ func (rl RunList) Run(log logr.Logger, tmClient kubernetes.Interface, namespace,
 					rl[i].Metadata.Testrun.ID = tr.Name
 				}
 				rl[i].Error = err
+				rl[i].Metadata.Retries = attempt - 1
 
 				if err == nil && tr.Status.Phase == tmv1beta1.PhaseStatusSuccess {
-					if flakeAttempt != 0 {
-						rl[i].Metadata.Flaked++
-					}
 					// testrun was successful, break retry loop
 					break
 				}
-				if flakeAttempt != maxFlakeAttempts {
+				if attempt < maxFlakeAttempts {
 					// clean status and name of testrun if it's failed to ignore it, since a retry will be initiated
+					log.V(3).Info(fmt.Sprintf("testrun failed, retry %d/%d. testrun", attempt, maxFlakeAttempts))
```

Comment:
I think we should make it a more important message
```suggestion
					log.Info(fmt.Sprintf("testrun failed, retry %d/%d. testrun", attempt, maxFlakeAttempts))
```

Reply:
done.